cli/flox/src/commands/search.rs

```diff
@@ -85,7 +87,13 @@ impl Search {
             lockfile.map(|p| p.try_into()).transpose()?,
         )?;
 
-        let (results, exit_status) = do_search(&search_params)?;
+        let (results, exit_status) = Dialog {
+            message: "Searching for packages...",
+            help_message: Some("This may take a while the first time you run it."),
+            typed: Spinner::new(|| do_search(&search_params)),
+        }
+        .spin_with_timeout(Duration::from_secs(5))?;
```

Comment:
Suggestion: You probably want a lower timeout on this. This timeout is for how long to wait before showing the spinner, correct? If so, probably set it to 500ms or 1s.

Reply:
I would have set no timeout at all but I think I read that was discussed. A timeout to show a spinner suggests to me that the system is already slow and struggling to start the spinner 🤷🏼‍♂️

I don't mind set it to .5 seconds or anywhere in between maybe @jennymahmoudi or @ghudgins have specific typical timeouts for this in mind